skimage/morphology/misc.py

```diff
@@ -92,6 +96,15 @@ def remove_small_objects(ar, min_size=64, connectivity=1, in_place=False):
     >>> d = morphology.remove_small_objects(a, 6, in_place=True)
     >>> d is a
     True
+    >>> e = morphology.remove_small_objects(a.astype(int), 6)
+    /usr/local/lib/python3.5/dist-packages/skimage/morphology/misc.py:135:
```

Comment:
This won't pass the test suite.  Let's remove this example.

Reply:
OK, that's a shame, I would have found this very useful when trying to figure out my problem...